packages/headless/src/features/search/search-actions-loader.ts

```diff
@@ -0,0 +1,55 @@
+import {AsyncThunkAction} from '@reduxjs/toolkit';
+import {AsyncThunkSearchOptions} from '../../api/search/search-api-client';
+import {Engine} from '../../app/engine';
+import {search} from '../../app/reducers';
+import {SearchAction} from '../analytics/analytics-utils';
+import {
+  executeSearch,
+  ExecuteSearchThunkReturn,
+  fetchMoreResults,
+  StateNeededByExecuteSearch,
+} from './search-actions';
+
+/**
+ * The search actions.
+ */
+export interface ISearchActions {
+  /**
+   * Executes a search query.
```

Comment:
Thoughts in the wording here? Since these are action _creators_, the more accurate description would be: "creates an action needed to execute a search query". But I felt this might be too pedantic,

Reply:
Generally, I'd agree with you that the longer description isn't needed, but I think here it would be valuable, mainly because you have to actually call the action creators. For a new user, they may think the call would look like `engine.dispatch(action)` instead of `engine.dispatch(action())`.

So my suggestion for this one to make it slightly less wordy would be `Creates an action that executes a search query.`. We can also more explicitly refer to action _creators_ in the usage documentation to make this clear.

Reply:
Okay, adjusting